Samples/excel-add-in-mail-merge/src/taskpane/taskpane.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <script type="text/javascript" src="https://appsforoffice.microsoft.com/lib/1/hosted/office.js"></script>
+    <script type="text/javascript" src="https://alcdn.msauth.net/browser/2.35.0/js/msal-browser.js"></script>
+    <script type="text/javascript"
+        src="https://cdn.jsdelivr.net/npm/@microsoft/microsoft-graph-client/lib/graph-js-sdk.js"></script>
+    <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <link rel="stylesheet"
+        href="https://static2.sharepointonline.com/files/fabric/office-ui-fabric-core/11.0.0/css/fabric.min.css" />
+    <link rel="stylesheet" href="taskpane.css" />
+    <title>Contoso Mail Merge</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+</head>
+
+<body>
+    <div class="container">
+        <h1 style="margin: 8px;">Mail Merge Example</h1>
+        <p>Please follow the steps outlined below to delve into the capabilities of Excel add-ins and unlock their
+            potential when
+            integrated with the Graph API, specifically in the context of a mail merger.</p>
+        <p></p>
+        <p><strong>Step 1: Generating Sample Data for Mail Merge </strong></p>
+        <p>Click on "Create Sample Data" to import the provided sample data into the spreadsheet.</p>
+        <button type="button" id="createSampleData">Create Sample Data</button>
+        <p><strong>Step 2: Verify the Email Template </strong></p>
+        <p>Review the email template below, ensuring that the parameters inside &lt;&lt;...&gt;&gt; align with the
+            column names in your current
+            spreadsheet. Click "Send Email" to dispatch emails based on the information in each data row.</p>
```

Comment:
Seems like "Send email" is already in step 3 in this list. Am I missing something? 
```suggestion
            spreadsheet. Select "Send Email" to dispatch emails based on the information in each data row.</p>
```